jwst/datamodels/slit.py

```diff
@@ -0,0 +1,163 @@
+from . import model_base
+from .image import ImageModel
+
+
+__all__ = ['SlitModel', 'SlitDataModel']
+
+
+class SlitDataModel(model_base.DataModel):
+    """
+    A data model for 2D images.
+
+    Parameters
+    ----------
+    init : any
+        Any of the initializers supported by `~jwst.datamodels.DataModel`.
+
+    data : numpy array
+        The science data.
+
+    dq : numpy array
+        The data quality array.
+
+    err : numpy array
+        The error array.
+
+    relsens : numpy array
+        The relative sensitivity table.
+
+    relsens2d: numpy array
```

Comment:
relsens2d is not included (not needed) in SlitModel, so remove it from the list here.